packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -71,7 +71,8 @@ const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
  */
 const generateImage = (image, alt, isAriaHidden, variant, handleFallback) => {
   const icon = generateImagePlaceholder(alt, isAriaHidden, variant);
-  return <TerraImage className={cx('image')} src={image} placeholder={icon} alt={alt} onError={handleFallback} />;
+  const imageFit = (variant === AVATAR_VARIANTS.USER || variant === AVATAR_VARIANTS.FACILITY) ? ImageFitTypes.COVER : undefined;
```

Comment:
Is there harm in setting the fit to 'cover' for the shared user variant as well, just to skip the check?

If there is a problem, how do we address it for the default, no consumer supplied image, avatar and facility variants?

Reply:
Fixed it here https://github.com/cerner/terra-core/pull/2494/commits/36742f3d9719b0fe00bbada1fcb416a338e7253e.